src/uu/du/src/du.rs

```diff
@@ -25,7 +25,6 @@ use std::os::unix::fs::MetadataExt;
 use std::os::windows::fs::MetadataExt;
 #[cfg(windows)]
 use std::os::windows::io::AsRawHandle;
-#[cfg(windows)]
 use std::path::Path;
```

Comment:
You removed the `cfg`, but now clippy is sad, because the import is not used on Ubuntu and Mac. I think the `#[cfg(windows)]` was correct, because it only seems to be used in functions that are tagged with `#[cfg(windows)]`.